README.md

```diff
@@ -145,32 +143,38 @@ This plugin creates device plugin endpoints based on the configurations given in
 
 ```json
 {
-    "resourceList":
-    [
-        {
-            "resourceName": "sriov_net_A",
-            "rootDevices": ["02:00.0", "02:00.2"],
-            "sriovMode": true,
-            "deviceType": "netdevice"
+    "resourceList": [{
+            "resourceName": "intel_sriov_netdevice",
+            "selectors": {
+                "vendors": ["8086"],
+                "devices": ["154c", "10ed"],
+                "drivers": ["i40evf", "ixgbevf"]
+            }
         },
         {
-            "resourceName": "sriov_net_B",
-            "rootDevices": ["02:00.1", "02:00.3"],
-            "sriovMode": true,
-            "deviceType": "vfio"
+            "resourceName": "intel_sriov_dpdk",
+            "selectors": {
+                "vendors": ["8086"],
+                "devices": ["154c", "10ed"],
+                "drivers": ["vfio-pci"]
+            }
         }
     ]
 }
 ```
 
 `"resourceList"` should contain a list of config objects. Each config object may consist of following fields:
 
-|     Field      | Required |                    Description                    |                       Type - Accepted values                        |         Example          |
-|----------------|----------|---------------------------------------------------|---------------------------------------------------------------------|--------------------------|
-| "resourceName" | Yes      | Endpoint resource name                            | `string` - must be unique and should not contain special characters | `"sriov_net_A"`          |
-| "rootDevices"  | Yes      | List of PCI address for a resource pool           | A list of `string` - in sysfs pci address format                    | `["02:00.0", "02:00.2"]` |
-| "sriovMode"    | No       | Whether the root devices are SRIOV capable or not | `bool` - true OR false[default]                                     | `true`                   |
-| "deviceType"   | No       | Device driver type                                | `string` - "netdevice"\|"uio"\|"vfio"                               | `"netdevice"`            |
+
+
+|     Field      | Required |        Description        |                       Type - Accepted values                        |                          Example/Accepted values                          |
+|----------------|----------|---------------------------|---------------------------------------------------------------------|---------------------------------------------------------------------------|
+| "resourceName" | Yes      | Endpoint resource name    | `string` - must be unique and should not contain special characters | "sriov_net_A"                                                             |
+| "selectors"    | No       | A map of device selectors | Each selector is a map of  `string` list.                           | "vendors": ["8086"], "devices": ["154c", "10ed"], "drivers": ["vfio-pci"] |
```

Comment:
what is the default behaviour if you don't use selectors? Does it select all or it won't select anyone?

Reply:
> what is the default behaviour if you don't use selectors? Does it select all or it won't select anyone?

A resourcPool with no selectors will include all network devices initially discovered by host.

Reply:
ok so it wasn't clear in the readme, unless I missed ...